Replace ember-cli-eyeglass with ember-cli-sass

mydea opened the pull request:
This replaces ember-cli-eyeglass with ember-cli-sass, as the former does not (currently) work on Windows, and this generally doesn't use anything eyeglass offers (as far as I could tell).

It also removes the unused ember-ajax dependency & bumps ember-engines.

efx commented:
@mydea thanks for working on this! This fixed a random node-sass bug on my machine. Could you rebase this on master and include just just the sass dependency changes?

Commit:
Replace ember-cli-eyeglass with ember-cli-sass

No eyeglass-specific features were used anyhow, and it lead to some build issues, especially on Windows.

mydea commented:
Sure, done!

efx commented:
@dgeb if you wouldn't mind merging / reviewing this before #48. I can update that PR for any conflicts.

dgeb commented:
@mydea thanks for the PR!

> if you wouldn't mind merging / reviewing this before #48. I can update that PR for any conflicts.

@efx ok, thanks - just merged this.
